lib/matplotlib/axes/_axes.py

```diff
@@ -5826,9 +5826,9 @@ def pcolor(self, *args, shading=None, alpha=None, norm=None, cmap=None,
             The coordinates of the corners of quadrilaterals of a pcolormesh::
 
                 (X[i+1, j], Y[i+1, j])       (X[i+1, j+1], Y[i+1, j+1])
-                                      +-----+
-                                      |     |
-                                      +-----+
+                                      ┌─────┐
```

Comment:
I kind of like identifying the corners in some way. Are you able to change it to `*---*` or some other identifier to get a distinct point at the corner? Right now this is just a box in the render and somewhat hard to identify exactly what the quantity in the parentheses is referring to.

Reply:
I agree that this one and the other one is this file should as they were.

Reply:
I sort of like the third one.

Reply:
I also like the circles. It could be a bit confusing with the connected bars at the top but not the vertical making me think I'm perhaps missing something, but not a major issue either.

Reply:
It's unfortunately due to the rectangular nature of the monospace characters with the circle being vertically centred, so not much can be done there.

Reply:
Unless, is there maybe a half horizontal line character so that we can stop the line early?

Reply:
Ah yes, there is such a thing:
```
                (X[i+1, j], Y[i+1, j])       (X[i+1, j+1], Y[i+1, j+1])
                                      ●╶───╴●
                                      │     │
                                      ●╶───╴●
                    (X[i, j], Y[i, j])       (X[i, j+1], Y[i, j+1])
```